diff --git a/dist/commonjs/index.js b/dist/commonjs/index.js
index 5a0ba2a87fb0365a959223919e82e6ac32dc49f8..2f9612ef4c21925b2d17df5ee3d3c84b679200b3 100644
--- a/dist/commonjs/index.js
+++ b/dist/commonjs/index.js
@@ -1032,4 +1032,8 @@ exports.minimatch.AST = ast_js_1.AST;
 exports.minimatch.Minimatch = Minimatch;
 exports.minimatch.escape = escape_js_1.escape;
 exports.minimatch.unescape = unescape_js_1.unescape;
+// Backward compatibility for call sites expecting require('minimatch') to be callable.
+const legacyDefault = Object.assign(exports.minimatch, exports);
+legacyDefault.default = exports.minimatch;
+module.exports = legacyDefault;
 //# sourceMappingURL=index.js.map
